email_template_qweb/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
```

Comment:
Do we need to change it to `The migration of this module from {{from_version}} to {{to_version}} was financially supported by Camptocamp
`

Reply:
ah, no need, because the file has already been existed